doc/src/guides/solving/solve-matrix-equation.md

```diff
@@ -0,0 +1,341 @@
+  convert it to matrix form, you can use {func}`~.linear_eq_to_matrix` and then
+  follow the procedures in this guide.
+
+## Solve a Matrix Equation
+
+Here is an example of solving a matrix equation with SymPy's
+{meth}`sympy.matrices.matrices.MatrixBase.solve`. We use the standard matrix
+equation formulation $Ax=b$ where
+- $A$ is the matrix representing the coefficients in the linear equations
+- $x$ is the column vector of unknowns to be solved for
+- $b$ is the column vector of constants, where each row is the value of an
+  equation
+
+```py
+>>> from sympy import init_printing
+>>> init_printing(use_unicode=True)
+```
+
+```py
+>>> from sympy import symbols
+>>> from sympy.matrices import Matrix
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c,d], [1, -e]])
+>>> A
+⎡c  d ⎤
+⎢     ⎥
+⎣1  -e⎦
+>>> b = Matrix([2, 0])
+>>> b
+⎡2⎤
+⎢ ⎥
+⎣0⎦
+>>> A.solve(b)
+⎡  2⋅e  ⎤
+⎢───────⎥
+⎢c⋅e + d⎥
+⎢       ⎥
+⎢   2   ⎥
+⎢───────⎥
+⎣c⋅e + d⎦
+```
+
+## Guidance
+
+### Matrix Must Be Square
+
+The matrix $A$ must be square to represent a system of linear equations with the
+same number of unknowns as equations. If not, SymPy will give the error
+``ShapeError: `self` and `rhs` must have the same number of rows.``
+
+### Solving Several Matrix Equations With the Same Matrix
+
+If you need to repeatedly solve matrix equations with the same matrix $A$ but
+different constant vectors $b$, it is more efficient to use one of the following
+methods: [LU decomposition](https://en.wikipedia.org/wiki/LU_decomposition) via
+{meth}`~sympy.matrices.matrices.MatrixBase.LUdecomposition`
+
+```py
+>>> from sympy import symbols, Matrix, eye, simplify
+>>> c, d, e = symbols("c, d, e")
+>>> A = Matrix([[c,d], [1, -e]])
+>>> A
+⎡c  d ⎤
+⎢     ⎥
+⎣1  -e⎦
+>>> L, U, perm = A.LUdecomposition()
+>>> L
+    ⎡1  0⎤
+    ⎢    ⎥
+    ⎣c  1⎦
+>>> U
+    ⎡1    -e   ⎤
+    ⎢          ⎥
+    ⎣0  c⋅e + d⎦
+>>> perm
+[[0, 1]]
+>>> b = Matrix([2, 0])
+>>> b
+    ⎡2⎤
+    ⎢ ⎥
+    ⎣0⎦
+>>> b2 = Matrix([4, 0])
+>>> b2
+    ⎡4⎤
+    ⎢ ⎥
+    ⎣0⎦
+>>> P = eye(A.rows).permuteFwd(perm)
+>>> P
+    ⎡0  1⎤
+    ⎢    ⎥
+    ⎣1  0⎦
+>>> y = L.solve(P*b) # Step-by-step approach, step 1
+>>> y
+    ⎡0⎤
+    ⎢ ⎥
+    ⎣2⎦
+>>> U.solve(y) # Step-by-step approach, step 2
+    ⎡  2⋅e  ⎤
+    ⎢───────⎥
+    ⎢c⋅e + d⎥
+    ⎢       ⎥
+    ⎢   2   ⎥
+    ⎢───────⎥
+    ⎣c⋅e + d⎦
+>>> solution = U.solve(L.solve(P*b)) # One-line approach
+>>> solution
+    ⎡  2⋅e  ⎤
+    ⎢───────⎥
+    ⎢c⋅e + d⎥
+    ⎢       ⎥
+    ⎢   2   ⎥
+    ⎢───────⎥
+    ⎣c⋅e + d⎦
+>>> simplify(A * solution) # Demonstrate that solution is correct
+    ⎡2⎤
+    ⎢ ⎥
+    ⎣0⎦
+>>> solution2 = U.solve(L.solve(P*b2)) # Repeating one-line approach for b2
+>>> solution2
+    ⎡  4⋅e  ⎤
+    ⎢───────⎥
+    ⎢c⋅e + d⎥
+    ⎢       ⎥
+    ⎢   4   ⎥
+    ⎢───────⎥
+    ⎣c⋅e + d⎦
+>>> simplify(A * solution2) # Demonstrate that solution2 is correct
+    ⎡4⎤
+    ⎢ ⎥
+    ⎣0⎦
+```
+
+or, likely slower, compute the inverse matrix using
```

Comment:
This is almost always slower and for larger matrices, significantly so. I recommend including a warning that states this. Using `inv()` is likely always the wrong choice for solving linear systems if there is any desire for efficient computation.